release/NativeEditPlugin/Plugins/iOS/EditBox_iOS.m

```diff
@@ -482,13 +481,14 @@ -(bool) isFocused
     return editView.isFirstResponder;
 }
 
--(void) showKeyboard:(bool)isShow
+-(void) hideKeyboard
 {
-    [viewController.view endEditing:(isShow ? YES : NO)];
```

Comment:
`viewController.view` is no longer the correct view with Unity 2018.1 and this method was rather hacky anyway.
Now we are using the proper method, which is calling `resignFirstResponder` on the button.